### ISSUE TITLE
Improvement : enable / disable features in config file or configuration page

### ISSUE THREAD

**Issue by hugoheitz** [seblucas/cops#328]
As the functions increase, it will become difficult to handle in the same interface.
It would be great to be able to enable / disable them in config files or even better in the configuration page.

For example, I know I wont use the direct read function, I would like to be able to disable it.

![cops_fiche_livre_avec_lien_mail_1](https://cloud.githubusercontent.com/assets/972023/23699351/1e6fece0-03ef-11e7-926b-5578933da6db.png)

In the configuration page :  
http://cops-demo.slucas.fr/index.php?page=19

it would be great to have the functions (enabled by default) and the capacity to enable or disable at will.

**Comment by horus68** [comment]
Agree on a config for more parameters.
Side note: for me the direct read icon its very useful for a quick preview or when showing the library to someone else so I can open a book without having to include it in my reading app just to open it.

**Comment by hugoheitz** [comment]
I agree, the direct read functionality is useful when you just want to show a part of the book to someone without having to import it.

My point is here to be able to easily activate / deactivate it.
